test/conformance/walk.go

```diff
@@ -85,13 +87,16 @@ func (v *visitor) convertToConformanceData(at *ast.BasicLit) {
 	cd.Description = ""
 	for _, line := range lines {
 		line = strings.TrimSpace(line)
-		if strings.HasPrefix(line, "Testname:") {
-			line = strings.TrimSpace(line[9:])
-			cd.TestName = line
+		if sline := regexp.MustCompile("Testname\\s*:\\s*").Split(line, -1); len(sline) == 2 {
+			cd.TestName = sline[1]
 			continue
 		}
-		if strings.HasPrefix(line, "Description:") {
-			line = strings.TrimSpace(line[12:])
+		if sline := regexp.MustCompile("Release\\s*:\\s*").Split(line, -1); len(sline) == 2 {
+			cd.Release = sline[1]
+			continue
+		}
+		if sline := regexp.MustCompile("Description\\s*:\\s*").Split(line, -1); len(sline) == 2 {
+			line = sline[1]
 		}
 		cd.Description += line + "\n"
```

Comment:
Are we concerned that newer test files with older walk code will treat newer fields as descriptions?

Reply:
Not really, we just add every other line in the comment block under the description after extracting the lines that has Testname and Version information.